### ISSUE TITLE
Item selection moves forward by a few items when modal opens (as if you were hitting tab)

### ISSUE THREAD

**Issue by lapinvert** [lwouis/alt-tab-macos#3592]
**Describe the bug**

I use command + tab to open the app selector layover, at which point the app that is supposed to be shown as selected is the previous one that I visited. But as I have released the tab key and I am still pressing the command one in order to have the time to see the apps and select the one I want to toggle, the selector sometimes moves forward by itself a little and then stops.

As if there was a few extra tab strokes from my end.

It would move by one item, two, or three most of the time. And in some rare case it would keep moving forward until the end, until the last item.

In other words it behaves AS IF I was hitting the "tab" key again and again, which I am not.

It doesn't do it each time though it is like almost half the time and this is very annoying since the app selection becomes unexpected.

**Your environment**

* AltTab version: Version 6.72.0
* macOS version: Sonoma 14.4

**Comment by lwouis** [comment]
https://github.com/lwouis/alt-tab-macos/issues/3117
